packages/hardhat/contracts/GardensTemplate.sol

```diff
@@ -9,62 +9,16 @@ import {ITollgate as Tollgate} from "./external/ITollgate.sol";
 import {
     IConvictionVoting as ConvictionVoting
 } from "./external/IConvictionVoting.sol";
-import "@1hive/apps-brightid-register/contracts/BrightIdRegister.sol";
 import "./external/Agreement.sol";
 import "./external/DisputableVoting.sol";
```

Comment:
We shouldn't use this approach actually. We should import all contracts using their own name without using "as" everywhere because BlockScout wouldn't verify this contract until I removed them.